# -*- coding: utf-8 -*-
from exception import PyLibsException


class NetworkException(PyLibsException):
    pass